src/library/Uno.Themes.WinUI.Markup/Theme/PasswordBox.cs

```diff
@@ -0,0 +1,230 @@
+
+						[ResourceKeyDefinition(typeof(Brush), "FilledPasswordBoxBorderBrushDisabled")]
+						public static ResourceValue<Brush> Disabled => new("FilledPasswordBoxBorderBrushDisabled", true);
+					}
+
+					public static class Foreground
+					{
+						[ResourceKeyDefinition(typeof(Brush), "FilledPasswordBoxForeground")]
+						public static ResourceValue<Brush> Default => new("FilledPasswordBoxForeground", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FilledPasswordBoxForegroundPointerOver")]
+						public static ResourceValue<Brush> PointerOver => new("FilledPasswordBoxForegroundPointerOver", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FilledPasswordBoxForegroundFocused")]
+						public static ResourceValue<Brush> Focused => new("FilledPasswordBoxForegroundFocused", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FilledPasswordBoxForegroundDisabled")]
+						public static ResourceValue<Brush> Disabled => new("FilledPasswordBoxForegroundDisabled", true);
+					}
+
+					public static class Placeholder
+					{
+						public static class Foreground
+						{
+							[ResourceKeyDefinition(typeof(Brush), "FilledPasswordBoxPlaceholderForeground")]
+							public static ResourceValue<Brush> Default => new("FilledPasswordBoxPlaceholderForeground", true);
+
+							[ResourceKeyDefinition(typeof(Brush), "FilledPasswordBoxPlaceholderForegroundPointerOver")]
+							public static ResourceValue<Brush> PointerOver => new("FilledPasswordBoxPlaceholderForegroundPointerOver", true);
+
+							[ResourceKeyDefinition(typeof(Brush), "FilledPasswordBoxPlaceholderForegroundFocused")]
+							public static ResourceValue<Brush> Focused => new("FilledPasswordBoxPlaceholderForegroundFocused", true);
+
+							[ResourceKeyDefinition(typeof(Brush), "FilledPasswordBoxPlaceholderForegroundDisabled")]
+							public static ResourceValue<Brush> Disabled => new("FilledPasswordBoxPlaceholderForegroundDisabled", true);
+						}
+					}
+
+					public static class RevealButton
+					{
+						public static class Foreground
+						{
+							[ResourceKeyDefinition(typeof(Brush), "FilledPasswordBoxRevealButtonForeground")]
+							public static ResourceValue<Brush> Default => new("FilledPasswordBoxRevealButtonForeground", true);
+						}
+					}
+
+					public static class Typography
+					{
+						[ResourceKeyDefinition(typeof(FontFamily), "FilledPasswordBoxFontFamily")]
+						public static ResourceValue<FontFamily> FontFamily => new("FilledPasswordBoxFontFamily", true);
+#if WinUI
+						[ResourceKeyDefinition(typeof(FontWeights), "FilledPasswordBoxFontWeight")]
+						public static ResourceValue<FontWeights> FontWeight => new("FilledPasswordBoxFontWeight", true);
+#else
+						[ResourceKeyDefinition(typeof(FontWeight), "FilledPasswordBoxFontWeight")]
+						public static ResourceValue<FontWeight> FontWeight => new("FilledPasswordBoxFontWeight", true);
+#endif
```

Comment:
```suggestion
						[ResourceKeyDefinition(typeof(FontWeights), "FilledPasswordBoxFontWeight")]
						public static ResourceValue<FontWeights> FontWeight => new("FilledPasswordBoxFontWeight", true);
```

We only support WinUI, no need to use `#if uwp/winui` in code